packages/react-examples/src/react-charting/HorizontalBarChartWithAxis/HorizontalBarChartWithAxis.Error.Example.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { HorizontalBarChartWithAxis, IHorizontalBarChartWithAxisProps } from '@fluentui/react-charting';
+
+interface IHorizontalBarChartWithAxisState {
+  width: number;
+  height: number;
+  isCalloutselected: boolean;
+  useSingleColor: boolean;
+}
+
+export class HorizontalBarChartWithAxisErrorExample extends React.Component<
+  IHorizontalBarChartWithAxisProps,
+  IHorizontalBarChartWithAxisState
+> {
+  constructor(props: IHorizontalBarChartWithAxisProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    return <div>{this._basicExample()}</div>;
+  }
+
+  private _basicExample(): JSX.Element {
+    return (
+      <>
+        <div style={{ maxWidth: 600 }}>
+          <HorizontalBarChartWithAxis
+            culture={window.navigator.language}
+            chartTitle="Horizontal bar chart error example "
+            data={[]}
```

Comment:
What is the minimum and maximum size of the chart container in case of error/empty state